experimental/samples/runtime/dotnet/slack-adapter/slack-adapter/Dialogs/echo/generated/echo.lu.qna.dialog

```diff
@@ -0,0 +1,4 @@
+{
+    "$kind": "Microsoft.CrossTrainedRecognizerSet",
+    "recognizers": []
+}
```

Comment:
Same thing here, if we intend to primarily use in Composer, I would remove this and let Composer handle generating it directly.